bin/reth/src/args/payload_builder_args.rs

```diff
@@ -81,7 +98,7 @@ impl TypedValueParser for ExtradataValueParser {
                 format!(
                     "Payload builder extradata size exceeds {MAXIMUM_EXTRA_DATA_SIZE}bytes limit"
                 ),
-            ))
+            ));
```

Comment:
```suggestion
            ))
```

Reply:
Addressed in https://github.com/anton-rs/op-reth/pull/145